jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l

```diff
@@ -428,9 +618,58 @@ Args:
       (room-light-off :control-switchbot control-switchbot))
     ;; change the inflation_radius
     (restore-params)
+    (send-kitchen-mail)
     (setq success-battery-charging (equal (get-battery-charging-state) :charging))
     (and success-go-to-kitchen success-auto-dock success-battery-charging)))
 
+(defun report-start-go-to-kitchen-state (args)
+  (let ()
+    (report-start-go-to-kitchen-state)
+    (set-alist 'description "キッチンを見に行くよ" args)
+    (set-alist 'image (remove #\newline (base64encode (send *image* :serialize))) args)
+    t))
+
+
+(defun get-light-on-state (args)
+  "get-light-on-state"
+  (let ((light-on (get-light-on)))
+    (set-alist 'initial-light-on light-on args)
+    (set-alist 'description "電気がついているか確認したよ" args)
+    (set-alist 'image (remove #\newline (base64encode (send *image* :serialize))) args)
+    light-on))
+
+(defun report-light-on-state (args)
+  "report-light-on-state"
+  (let ()
+    (report-light-on-state)
+    (set-alist 'description "電気がついていたよ" args)
+    (set-alist 'image (remove #\newline (base64encode (send *image* :serialize))) args)
+    t))
+
+(defun room-light-on-state (args)
+  "room-light-on-state"
+  (let ((control-switchbot (cdr (assoc 'control-switchbot args))))
+    (report-light-off)
+    (room-light-on :control-switchbot control-switchbot)
+    (set-alist 'description "電気をつけたよ" args)
+    (set-alist 'image (remove #\newline (base64encode (send *image* :serialize))) args)
+    t))
+
+(defun move-to-sink-front-status (args)
+  (let* ((n-trial (cdr (assoc 'n-kitchen-trial args)))
+         (success (move-to-sink-front :n-trial n-trial)))
+    (setf (cdr (assoc 'success-go-to-kitchen args)) success)
+    (set-alist 'description "キッチンに向かったよ" argw)
+    (set-alist 'image (remove #\newline (base64encode (send *image* :serialize))) args)
+    success))
+
+(defun inspect-kitchen-state (args)
+  "inspect-kitchen"
+  (let ()
+    (inspect-kitchen :tweet (cdr (assoc 'tweet args)))
+    (set-alist 'description "キッチンの様子を見たよ" args)
```

Comment:
as I comment on https://github.com/jsk-ros-pkg/jsk_roseus/pull/716#issuecomment-1225047867, DESCRIPTION is the message from robot to users to tell how they thik and what they do. so 
```
(let ((label-names (inspect-kitchen ....))
      (set-alist 'description (format nil "キッチンの様子を見たよ。~Aがあったよ" label-names) args)

(defun notify-recognition (&key (location "kitchen"))
  ...
  label-names)
 ...

Reply:
Thank you very much for advice.
I add notify-text in description and write like this. [ba5fc8f](https://github.com/jsk-ros-pkg/jsk_robot/pull/1568/commits/ba5fc8f0d4bdcbdbfd18b574c96ea98cd8513af9)
```
キッチンの様子を見たよ．cupがあったよ
[ IMAGE ]
```

Reply:
Like this, this is tracshcan example, though.
![B7DB04F2-F5E5-4259-9025-EA59CD487257](https://user-images.githubusercontent.com/67531577/186387999-f0755e1f-17bb-40ae-b037-9a395b7089a6.png)